python/lsst/rapid/analysis/nightReport.py

```diff
@@ -354,9 +357,9 @@ def _safeListArg(arg):
 
 
 if __name__ == '__main__':
```

Comment:
There was discussion above about moving these things to a `bin/` script.

Reply:
Will be moving all of these to some butler-driven tests now! 🙂